app/invocation/invocation_exec_log_card.tsx

```diff
@@ -257,9 +255,9 @@ export default class InvocationExecLogCardComponent extends React.Component<Prop
                     <input
                       id="direction-asc"
                       checked={this.state.direction == "asc"}
-                      onChange={this.handleInputChange.bind(this)}
+                      onChange={this.handleSortDirectionChange.bind(this)}
                       value="asc"
-                      name="direction"
+                      name="exec-log-direction"
```

Comment:
in other "name" fields in the past I think we've used `camelCase`

```suggestion
                      name="execLogDirection"
```